tests/playbooks/tests_eth_pci_address_match.yml

```diff
@@ -0,0 +1,70 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  tasks:
+    - import_role:
+        name: linux-system-roles.network
+      vars:
+        network_connections:
+          - name: pseudo-pci-test-only
+            persistent_state: present
+            type: ethernet
+            # `pci-0001:00:00.0` is a pseudo PCI address,
+            # which used only for testing purpose
+            match:
+              path:
+                - pci-0001:00:00.0
+            ip:
+              dhcp4: no
+              address:
+                - 192.0.2.3/24
+    - block:
+        - name: Get match.path setting for connection
+          command: nmcli -f match.path connection show pseudo-pci-test-only
+          register: match_path
+          ignore_errors: yes
+          changed_when: false
+
+        - name: Assert that the connection profile contains the specified PCI
+            address
+          assert:
+            that:
+              - "'pci-0001:00:00.0' in match_path.stdout"
+            msg: The connection profile does not contain the specified PCI
+              address
+
+        - name: "Remove the PCI address from match path"
+          import_role:
+            name: linux-system-roles.network
+          vars:
+            network_connections:
+              - name: pseudo-pci-test-only
+                type: ethernet
+                interface_name: ""
```

Comment:
Please add a comment why `interface_name` needs to be set to empty, here.

Reply:
Comment was added.